exchange/exchange.go

```diff
@@ -518,6 +524,20 @@ func (e *exchange) getAllBids(
 	return adapterBids, adapterExtra, bidsFound
 }
 
+func applyFPD(fpdData map[openrtb_ext.BidderName]*openrtb_ext.ORTB2, bidReq *openrtb2.BidRequest, bidderName openrtb_ext.BidderName) {
```

Comment:
Please add unit tests for this function

Reply:
Please consider moving this func to utils.go where it is used.

Reply:
Agree, moved and unite test added